### ISSUE TITLE
[Failure] [master] ClientServerSpec: low-level HTTP infra should complete a simple request/response cycle

### ISSUE THREAD

**Issue by ennru** [akka/akka-http#3122]
Timeout.

`akka.http.scaladsl.ClientServerSpec.The low-level HTTP infrastructure should properly complete a simple request/response cycle using Http.singleRequest`

Akka 2.5
Scala 2.12

```
java.lang.AssertionError: assertion failed: timeout (3 seconds) during expectMsg while waiting for OnComplete
	at scala.Predef$.assert(Predef.scala:223)
	at akka.testkit.TestKitBase.expectMsg_internal(TestKit.scala:418)
	at akka.testkit.TestKitBase.expectMsg(TestKit.scala:395)
	at akka.testkit.TestKitBase.expectMsg$(TestKit.scala:395)
	at akka.testkit.TestKit.expectMsg(TestKit.scala:923)
	at akka.stream.testkit.TestSubscriber$ManualProbe.expectComplete(StreamTestKit.scala:439)
	at akka.http.scaladsl.ClientServerSpec$$anon$7.$anonfun$new$50(ClientServerSpec.scala:504)
```

https://jenkins.akka.io:8498/job/akka-http-nightly/AKKA_VERSION=release-2.5,Node=akka-http-nightly-node,SCALA_VERSION=2.12.10,jdk=AdoptOpenJDK%208/lastCompletedBuild/testReport/akka.http.scaladsl/ClientServerSpec/The_low_level_HTTP_infrastructure_should_properly_complete_a_simple_request_response_cycle_using_Http_singleRequest/

**Comment by ignasi35** [comment]
https://jenkins.akka.io:8498/job/akka-http-nightly-release-10.1/957/AKKA_VERSION=default,Node=akka-http-nightly-node,SCALA_VERSION=2.12.10,jdk=AdoptOpenJDK%208/testReport/junit/akka.http.scaladsl/ClientServerSpec/The_low_level_HTTP_infrastructure_should_properly_complete_a_simple_request_response_cycle/

```
java.net.BindException: Cannot assign requested address
	at sun.nio.ch.Net.bind0(Native Method)
	at sun.nio.ch.Net.bind(Net.java:433)
	at sun.nio.ch.Net.bind(Net.java:425)
	at sun.nio.ch.ServerSocketChannelImpl.bind(ServerSocketChannelImpl.java:220)
	at sun.nio.ch.ServerSocketAdaptor.bind(ServerSocketAdaptor.java:85)
	at sun.nio.ch.ServerSocketAdaptor.bind(ServerSocketAdaptor.java:78)
	at akka.testkit.SocketUtil$.$anonfun$temporaryServerAddresses$1(SocketUtil.scala:97)
	at scala.collection.generic.GenTraversableFactory.fill(GenTraversableFactory.scala:92)
	at akka.testkit.SocketUtil$.temporaryServerAddresses(SocketUtil.scala:81)
	at akka.testkit.SocketUtil$.temporaryServerAddress(SocketUtil.scala:74)
	at akka.testkit.SocketUtil$.temporaryServerHostnameAndPort(SocketUtil.scala:106)
	at akka.http.scaladsl.ClientServerSpec$TestSetup.<init>(ClientServerSpec.scala:818)
	at akka.http.scaladsl.ClientServerSpec$$anon$6.<init>(ClientServerSpec.scala:450)
	at akka.http.scaladsl.ClientServerSpec.$anonfun$new$46(ClientServerSpec.scala:450)
	at akka.stream.testkit.scaladsl.StreamTestKit$.assertAllStagesStopped(StreamTestKit.scala:32)
	at akka.stream.testkit.Utils$.assertAllStagesStopped(Utils.scala:21)
	at akka.http.scaladsl.ClientServerSpec.$anonfun$new$45(ClientServerSpec.scala:449)
	at org.scalatest.OutcomeOf.outcomeOf(OutcomeOf.scala:85)
	at org.scalatest.OutcomeOf.outcomeOf$(OutcomeOf.scala:83)
	at org.scalatest.OutcomeOf$.outcomeOf(OutcomeOf.scala:104)
```

**Comment by jrudolph** [comment]
Fixed by #3460.
